pos_default_payment_method/models/pos_config.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2017-TODAY Camptocamp SA (<http://www.camptocamp.com>).
+# @author: Simone Orsi (https://twitter.com/simahawk)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import fields, models
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    default_payment_method_id = fields.Many2one(
+        comodel_name='account.journal',
+        string='Default payment method',
+        domain=[('journal_user', '=', True),
+                ('type', 'in', ['bank', 'cash'])],
+    )
```

Comment:
please set a constraint on journals pos config, to avoid bad settings.

Reply:
https://github.com/OCA/pos/pull/211/commits/62a337ac2840aa6edee9021cb04aa61618f2a5e9 is this what you intended?